app/assets/javascript/dropdown_select_all.js

```diff
@@ -0,0 +1,40 @@
+(function($) {
+  "use strict";
+  window.GOVUK = window.GOVUK || {};
+
+  function DropdownSelectAll(options) {
+    this.$dropdowns = $(options.$el);
```

Comment:
I've assigned the outside `this` to `that` to match what happens in `govuk_admin_template`.